### DISCUSSION TITLE
⛔ 🔗 Containers and sections at the same time (Duplicate of #20243)

### DISCUSSION THREAD

**Discussion by steveindzine** [elementor/elementor#21209] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

As a developer I understand containers, they make a lot of sense, but my users don't really understand them and get confused. I would like to use both at the same time, not one or the other. I want to reduce confusion but also have the freedom to use the advanced layout options.

### Describe the solution you'd like

I want my users to be able to carry on using sections and columns, but at the same time I want to be able to make more advanced layouts using containers.

**Comment by black-eye** [comment]
Yes, I'd like to have them both too. Please give us the option to keep the sections and columns in the left panel to use them together with containers.

**Comment by perks98** [comment]
+1 I agree.

**Comment by nicholaszein** [comment]
Hello @steveindzine,

Thank you for submitting a feature request! 🙏 

### ⛔ 🚀 After reviewing your request, we have decided not to add this to our roadmap.  

Sections will soon be replaced by Containers. Older sites with Sections and Columns will remain to function normally, and you can convert them to containers in your own time or leave them as they are, whatever suits your needs. You will be able to edit and duplicate Sections and Columns as many times as you want, but moving forward (in the near future) the Container is going to be used as part of the Core Elementor plugin and set as the default building element for new websites.

Feel free to contact us again with other feature suggestions. 😉 

Best regards
